packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -559,8 +559,9 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
      * Executes a raw query. Always returns a number
      */
     private $executeRawInternal(
-      runInTransaction: boolean,
```

Comment:
Note that runInTransaction is documented in docs 
https://sourcegraph.com/search?q=context:global+repo:github.com/prisma/+runInTransaction+repo:%5Egithub%5C.com/prisma/docs%24+&patternType=literal

So can technically be a breaking change in API?

Reply:
Or is it just changed inside internals here?

Reply:
Looks ok from https://github.com/prisma/prisma/pull/10764/files#diff-eca352fdbc50201808d520041da8d6a977697fc51eb82750889c0c1ff670a904R38-R46

Reply:
GH does not load that link. What makes the removable ok @Jolg42 ?